ereol_inspiration/ereol_inspiration.features.field_instance.inc

```diff
@@ -78,13 +118,15 @@ function ereol_inspiration_field_default_field_instances() {
       'module' => 'paragraphs',
       'settings' => array(),
       'type' => 'paragraphs_embed',
-      'weight' => 41,
+      'weight' => 5,
     ),
   );
 
   // Translatables
   // Included for use with string extractors like potx.
+  t('Category query');
   t('Elementer');
+  t('Søgestreng for hovedkategorien');
```

Comment:
All exported strings should be in english.

Reply:
Fixed.